pkg/kubernetes/kubernetes.go

```diff
@@ -104,6 +104,12 @@ func (h *Helper) Drain(node string) error {
 	for _, pod := range pods.Items {
 		go func(p corev1.Pod) {
 			defer wg.Done()
+			for _, ref := range p.ObjectMeta.OwnerReferences {
+				if ref.Kind == "DaemonSet" {
```

Comment:
DaemonSet managed pods are impossible to get rid of as long as the Kubelet is running. Skip them to speed up upgrades and prevent any thrashing that could potentially fail the upgrade.